geno_project.lua

```diff
@@ -1,11 +1,48 @@
 local p = premake
+local project = p.project
+local tree = p.tree
 local m = { }
 
+-- Alphabetic Compare
+local function alphabeticCompare( a, b )
+	if not a and b then
+		return true
+	elseif not b and a then
+		return false
+	elseif not a and not b then
+		return true
+	end
```

Comment:
In which circumstances are the arguments null?

Reply:
Oh lol that's not supposed to be there, I was trying to figure out why doing `str[ i ]` was returning null which happens due to strings in lua not being an array of characters.